### PR TITLE
added improvements to nim on vertex notebook

### PR DESCRIPTION
**Pull request by juanpabloguerra16**

### CODE REVIEW

File: cloud-service-providers/google-cloud/vertexai/python/nim-vertexai.ipynb [hunk 1]
```diff
@@ -123,7 +105,7 @@
    },
    "outputs": [],
    "source": [
-    "import google.cloud.aiplatform as aiplatform\n",
+    "import google.cloud.aiplatform as aiplatform # redundant?\n",
```
Comment (juanpabloguerra16) [inline cloud-service-providers/google-cloud/vertexai/python/nim-vertexai.ipynb:108]: @FortunaZhang I'm not sure if importing this package here makes sense as there's another cell after importing:

`from google.cloud import aiplatform`

it seems a bit redundant but please lmk your thoughts

Comment (FortunaZhang) [reply cloud-service-providers/google-cloud/vertexai/python/nim-vertexai.ipynb:108]: Yes this is same as the other import below.  

from google.cloud import aiplatform
aiplatform.init(project=project_id, location=region, staging_bucket=bucket_url)

**Review by FortunaZhang — COMMENTED** [review]
Thanks for the comments. Added some explanations and suggestions.

File: cloud-service-providers/google-cloud/vertexai/python/nim-vertexai.ipynb [hunk 2]
```diff
@@ -571,7 +530,7 @@
    "id": "5b3a6114-4105-4c86-88e9-246d6b43ed09",
    "metadata": {},
    "source": [
-    "### Run NIM Container on the CLI"
+    "### Run NIM Container Within Interface"
    ]
   },
   {
```
Comment (FortunaZhang) [inline cloud-service-providers/google-cloud/vertexai/python/nim-vertexai.ipynb:536]: Do we want to add 'export' commands to Terminal version?

    export container_name=
    export NGC_API_KEY=
    export selected_profile=
    export local_nim_cache= or $LOCAL_NIM_CACHE:/opt/nim/.cache

Comment (FortunaZhang) [reply cloud-service-providers/google-cloud/vertexai/python/nim-vertexai.ipynb:536]: Other changes look good.

**Review by FortunaZhang — APPROVED** [review]
Approve the improvements, thank you

**Pull request merged**